src/app/portlets/dot-edit-page/content/shared/iframe-edit-mode.css.ts

```diff
@@ -139,14 +139,19 @@ export const EDIT_PAGE_CSS = `
         opacity: 1;
     }
 
-     .dotedit-contentlet__toolbar button {
+    .dotedit-contentlet__toolbar button {
         margin-right: 8px;
```

Comment:
create variable equivalent to `$basic-padding` in this file ?

Reply:
We should.